tests/plugins/a11yhelp/a11yhelp.js

```diff
@@ -0,0 +1,146 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: a11yhelp,dialog*/
+
+( function() {
+	'use strict';
+
+	bender.editor = {};
+
+	var tests = {
+		_registerCommand: function( commandName, commandDefinition, keystroke ) {
+			var editor = this.editor;
+
+			if ( !commandDefinition ) {
+				commandDefinition = {
+					exec: function() {},
+					label: 'Test Label.'
+				};
+			}
+
+			if ( !keystroke ) {
+				keystroke = CKEDITOR.ALT + 65; // Alt + A
+			}
+
+			if ( !editor.getCommand( commandName ) ) {
+				editor.addCommand( commandName, commandDefinition );
+				editor.setKeystroke( keystroke, commandName );
+			}
+		},
+
+		_createEditor: function( config, fn ) {
+			if ( bender.editor ) {
+				bender.editor.destroy();
+			}
+
+			var tc = this;
+
+			bender.editorBot.create( { config: config }, function( bot ) {
+				bender.editor = tc.editor = bot.editor;
+				tc.editorBot = bot;
+				fn.call( tc );
+			} );
+		},
+
+
+		'test representKeystroke': function() {
+			this._createEditor( {}, function() {
+				var editor = this.editor;
+				assert.areSame( '<kbd><kbd>Alt</kbd>+<kbd>0</kbd></kbd>', CKEDITOR.plugins.a11yhelp.representKeystroke( editor, CKEDITOR.ALT + 48 ) );
+			} );
+		},
+
+		'test keystrokeEntry event': function() {
```

Comment:
There are still some leftovers of `keystrokeEntry` event in this file and also in `tests/plugins/a11yhelp/manual/commandstable` manual test (which fails now).